clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/ApplicationEventProcessor.java

```diff
@@ -422,6 +439,33 @@ private void process(final ShareAcknowledgementCommitCallbackRegistrationEvent e
         manager.setAcknowledgementCommitCallbackRegistered(event.isCallbackRegistered());
     }
 
+    private void process(final StreamsOnTasksRevokedCallbackCompletedEvent event) {
+        if (!requestManagers.streamsMembershipManager.isPresent()) {
```

Comment:
Maybe use `requestManagers.streamsMembershipManager.isEmpty()` instead here and below.